routes/index.js

```diff
@@ -105,4 +107,21 @@ const routes = [
   },
 ];
 
-module.exports = routes;
+function stripUnknownParamsOnProduction(route) {
+  const inProduction = (process.env.NODE_ENV === 'production');
+  const result = _.cloneDeep(route);
```

Comment:
Why not work directly on `route`, besides good Node habits? I'm just curious if you observed any weird side effects.